dask_geopandas/core.py

```diff
@@ -336,6 +337,33 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def morton_distance(self, p=15):
+
+        """
+
```

Comment:
TODO: expand the docstring (quick explanation, mention "also known as", maybe link to wiki page)

Reply:
Updated in 703b5b4

Reply:
@tastatham This is a top-level API function (only one in this PR) so this one needs a proper docstring since this will show up in the API docs (once they're done). We need a full docstring here. Everything else is internal. This even looks that you have removed part of the docstring from here.

Reply:
You can copy the docstring you added to `_morton_distance` here

Reply:
updated in 2c539e0